src/utils/bridge/eth/utils.ts

```diff
@@ -80,7 +80,7 @@ export const waitForApprovedRequest = async (tx: EthHistory): Promise<EthApprove
 
   subscription.unsubscribe();
 
-  return ethBridgeApi.getApprovedRequest(tx.hash as string);
+  return ethBridgeApi.getApprovedRequest(tx.hash as string) as Promise<EthApprovedRequest>;
```

Comment:
maybe it's slightly better

<img width="867" alt="image" src="https://github.com/sora-xor/polkaswap-exchange-web/assets/17688105/9b1e1d61-3f2d-4ed5-b519-cea86e1daf2a">